src/include/daos_fs.h

```diff
@@ -1136,12 +1135,14 @@ dfs_listxattr(dfs_t *dfs, dfs_obj_t *obj, char *list, daos_size_t *size);
 
 
 enum {
-	/** print the OIDS */
+	/** print the leaked OIDS */
 	DFS_CHECK_PRINT		= (1 << 0),
 	/** remove / punch the leaked objects */
 	DFS_CHECK_REMOVE	= (1 << 1),
-	/** add leaked oids to a "lost+found" directory */
-	DFS_CHECK_LINK_LF	= (1 << 2),
+	/** relink the leaked oids under "/lost+found" */
+	DFS_CHECK_RELINK	= (1 << 2),
```

Comment:
Does this introduce a potential compatibility issue?

Reply:
this is all new stuff that has been added to the feature branch and not in master, so that API and flags can still be changed whatever way we want.